apps/server/src/shared/domain/domainobject/board/board-composite.do.spec.ts

```diff
@@ -115,4 +143,17 @@ describe(`${BoardComposite.name}`, () => {
 			});
 		});
 	});
+
+	describe('getChildrenOfType', () => {
+		it('should return the children of the given type', () => {
+			const { parent, children } = setup();
+			const expectedChildren = [children[3].children[0], children[3]];
+
+			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+			// @ts-ignore
```

Comment:
is the ts-ignore necessary or can you use casting?

Reply:
I cant use cascading. The thing with that is i would need to change the AnyBoardDo type to accept those types. Let us talk.. maybe you have another idea for this :)